front-end/src/App.tsx

```diff
@@ -9,21 +9,21 @@ import InitialBanner from './components/InitialBanner';
 import PartnersBanner from './components/PartnersBanner';
 import TeamMembersBanner from './components/TeamMembersBanner';
 import FAQBanner from './components/FAQBanner';
-import RoadMapBanner from './components/RoadMapBanner';
 import PerkBanner from './components/PerkBanner';
 import './App.scss';
+import CollapsableRoadmapBanner from './components/CollapsableRoadmapBanner';
 
 library.add(fab, faBars, faTimes);
 
 function App() {
   return (
     <div className="App">
+      <AboutSurgeBanner />
```

Comment:
This should be above `TeamMemberBanner`. `InitialBanner` should be the first one.